### ISSUE TITLE
NVDA doesn't read title bar in web browsers

### ISSUE THREAD

**Issue by eltonfreak** [nvaccess/nvda#8740]
JAWS reads title bars when using up arrow and home NVDA doesn't even show it.
I can use NVDA+T but would prefer not to

**Comment by DrSooom** [comment]
Did you try to add another shortcut for reading the title bar via the Input Gestures Window? I'm not sure if ArrowUp+Home will works, but just try it.

PS: And please fill out the draft next time. thanks.

**Comment by jscholes** [comment]
The browser title bar is not a part of the page content, so unlike JAWS NVDA does not treat it as such.  As you point out, you can read the title bar with the command designated for this purpose.

If you prefer the JAWS-style behaviour, you are welcome to either create an add-on to replicate it, or post on the [NVDA Add-Ons Community](https://nvda-addons.groups.io/) to see if someone would be willing to create it for you.

**Comment by ehollig** [comment]
Closing because of above comment.
